CoreErrorReportView/release/modules/ext/docs/CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/error-report.md

```diff
@@ -0,0 +1,56 @@
+# Error Report
+
+<div style="text-align: center">
+    <img height=500 src="../ext/docs/CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/resources/errorReport.png" alt="Error Report" />
```

Comment:
This image would be more like what a user would see if there was an error displayed

Reply:
to add further, I would have the screenshot only show the error report (i.e. remove the rest of the application from it. It should simply demonstrate the view in action